src/test/java/uk/gov/hmcts/reform/orgrolemapping/helper/TestDataBuilder.java

```diff
@@ -343,8 +344,14 @@ public static RefreshJobEntity buildRefreshJobEntity() {
     }
 
     public static JudicialAccessProfile buildJudicialAccessProfile() {
-        return JudicialAccessProfile.builder().userId(id_1).roleId("8")
-                .baseLocationId("south-east")
+        return JudicialAccessProfile.builder()
+                .userId(id_1)
+                .roleId("84")
+                .contractTypeId("5")
+                .beginTime(ZonedDateTime.now(ZoneOffset.UTC).plusDays(1))
+                .endTime(ZonedDateTime.now(ZoneOffset.UTC).plusMonths(1))
+                .baseLocationId("1")
```

Comment:
why we are adding one month in end date and one day in begin date?

Reply:
its just for testing purposes to keep them separate